dashboards-observability/public/components/event_analytics/explorer/explorer.tsx

```diff
@@ -168,13 +173,11 @@ export const Explorer = ({
   liveTailTabIdRef.current = liveTailTabId;
   liveTailNameRef.current = liveTailName;
 
-  let minInterval = 'y';
   const findAutoInterval = (start: string = '', end: string = '') => {
-    if (start?.length === 0 || end?.length === 0 || start === end)
-      return ['d', [...TIME_INTERVAL_OPTIONS]];
     const momentStart = dateMath.parse(start)!;
     const momentEnd = dateMath.parse(end)!;
```

Comment:
could you also change this
```suggestion
    const momentEnd = dateMath.parse(end, { roundUp: true })!;
```

Reply:
Done in : [1776e8c](https://github.com/opensearch-project/observability/pull/1266/commits/1776e8cdbc4e0ca487c6519d80ce731dd8a70cee). @joshuali925 I also did this anywhere else I saw datemath.parse in endtime/lte and also removed duplicate code fragment. Can you double check? Also do we need to do anything for startTime?

Reply:
looks good, startTime by default rounds down so it's ok. this is for when user selects 'this week' or 'this month'